src/main/java/assignment1/exceptions/InvalidArguementsException.java

```diff
@@ -0,0 +1,8 @@
+package assignment1.exceptions;
+
+public class InvalidArguementsException extends Exception {
+
+  public InvalidArguementsException(String s) {
```

Comment:
if the class is declared public, then the default constructor is implicitly given the access modifier `public`